### DISCUSSION TITLE
Reach out to Multifan

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [NOAA-FIMS/collaborative_workflow#125] [General]
**Is your feature request related to a problem? Please describe.**
[Claudio-Castillo](https://github.com/Claudio-Castillo) would like join the FIMS conversation with regards to how [Multifan-CL](https://mfcl.spc.int/).

**Describe the solution you'd like**
Have a conversation with the developers / users of Multifan-CL

**Describe alternatives you've considered**

**Additional context**
Tagging data and movement are the two biggest priorities of Multifan-CL that are only marginally available in SS3.

**Comment by Claudio-Castillo** [comment]
Thanks @kellijohnson-NOAA , I will include in the loop @arni-magnusson, @Nick-Davies and @paulham44

**Comment by arni-magnusson** [comment]
Just a quick update, Paul Hamer's username is @Paul-Hamer.

There is interest here at SPC to experiment with a TMB module handling tag releases and recaptures. Is there already a worked example of what a FIMS TMB module might look like, or are we first in line?
